bindings/python/test/test_arrow.py

```diff
@@ -384,3 +388,183 @@ def test_find_decimal128(self):
         )
         table = find_arrow_all(coll, {}, schema=schema)
         self.assertEqual(table, expected)
+
+
+class TestNulls(unittest.TestCase):
```

Comment:
I'd suggest moving this class to `test/utils.py` and using `raise NotImplementedError` for the methods that need to be overridden.